IBAnimatable/AnimatableStackView.swift

```diff
@@ -146,11 +146,14 @@ open class AnimatableStackView: UIStackView, CornerDesignable, FillDesignable, B
   // MARK: - MaskDesignable
   open var maskType: MaskType = .none {
     didSet {
+      previousMaskType = oldValue
```

Comment:
Shall we put this line in `configureMask(previousMaskType: oldValue)`. Then we don't copy & paste it in all `Animatable*` classes. But it will have a problem of introducing breaking API for `configureMask()`. One workaround I can think of is to use optional and default value (`nil`) like

```
public func configureMask(previousMaskType: MaskType? = nil)
```

Then we don't even need `var previousMaskType: MaskType { get set }` any more.